eta-company-calendar-backend/src/main/java/hu/flowacademy/companycalendar/service/CommentService.java

```diff
@@ -0,0 +1,57 @@
+package hu.flowacademy.companycalendar.service;
+
+import hu.flowacademy.companycalendar.model.Comment;
+import hu.flowacademy.companycalendar.model.Meeting;
+import hu.flowacademy.companycalendar.model.User;
+import hu.flowacademy.companycalendar.model.dto.CommentDTO;
+import hu.flowacademy.companycalendar.repository.CommentRepository;
+import hu.flowacademy.companycalendar.repository.MeetingRepository;
+import hu.flowacademy.companycalendar.repository.UserRepository;
+import lombok.AllArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+import org.springframework.web.server.ResponseStatusException;
+
+import javax.transaction.Transactional;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+@Transactional
+@AllArgsConstructor
+public class CommentService {
+
+    private final CommentRepository commentRepository;
+    private final UserRepository userRepository;
+    private final MeetingRepository meetingRepository;
+
+    public List<Comment> findAll() {
+        return commentRepository.findAll();
+    }
+
+    public Comment findOne(Long id) {
+        return commentRepository.findById(id).orElseThrow();
+    }
+
+    public Comment saveComment(CommentDTO commentDTO) {
+        Comment comment = commentDTO.toEntity(
+                userRepository.findById(commentDTO.getUserId()).orElseThrow(() -> new ResponseStatusException(HttpStatus.NOT_FOUND)),
+                meetingRepository.findById(commentDTO.getMeetingId()).orElseThrow(() -> new ResponseStatusException(HttpStatus.NOT_FOUND))
+        );
+        return commentRepository.save(comment);
+    }
+
+    public Comment updateComment(CommentDTO commentDTO) {
+        Comment comment = findOne(commentDTO.getId());
+        comment.setContent(commentDTO.getContent());
+        return commentRepository.save(comment);
+    }
+
+    public void deleteComment(Long id) {
+        commentRepository.deleteById(id);
+    }
+
+    public List<Comment> findAllByMeetingId(Long meetingId) {
+        return commentRepository.findAll().stream().filter(m -> m.getId() == meetingId).collect(Collectors.toList());
```

Comment:
**TL;DR használjunk equals()-t, == helyett.**

A boxolt Longokat a Java egy bizonyos intervallumban cacheli (így mindig ugyanazt a példányt kapod meg ha többször példányosítanád ugyanazt a Longot, ~ singleton), emiatt ebben az esetben működik az ==, DE csak akkor, ha a szám az adott intervallumban van.
Dupla egyenlőséget csakis primitív típusok esetében használjunk. Objektumoknál **mindig** az `equals()` metódust használjuk.
Olvasmány ehhez: https://stackoverflow.com/questions/20541636/comparing-boxed-long-values-127-and-128